### PR TITLE
Student dropdown

### PR DESCRIPTION
**Pull request by judan**
Finished the dropdown. It's showing student names from the database now. 
*Note: Whoever's doing the read button will probably need to use the info (student id) in this component whenever the user clicks.

### CODE REVIEW

**Comment by hammour** [comment]
👍  looks good to me!

File: public/scripts/components/pages/Read.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+
+export default React.createClass({
+	render: function() {
+		return (
+			<section className="read-story">
+				<h1>Read Story</h1>
+			</section>
+		);
+	}
+});
```
Comment (alarner) [inline public/scripts/components/pages/Read.js:34]: You shouldn't be changing the Read component for this ticket. Why does it look like this file is modified?

File: migrations/20160422160813_create_story.js [hunk 2]
```diff
@@ -9,7 +9,9 @@ exports.up = function(knex, Promise) {
 		t.string('title').notNull();
 		t.string('author').nullable();
 		t.string('coverImage').nullable();
+		t.text('description').nullable();
 		t.enu('category', ['fable']).nullable();
+
```
Comment (alarner) [inline migrations/20160422160813_create_story.js]: You shouldn't have to modify this migration for this ticket. Undo your changes to this file please

File: public/scripts/components/sub-components/StudentDropdown.js [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+				<select name="mydropdown">
+					<option value="Pick">Pick a student</option>
+					{allStudents}
+				</select>
+			</div>
+		</div>
+		);
+	}
+});
+
```
Comment (alarner) [inline public/scripts/components/sub-components/StudentDropdown.js]: In order to be able to get the selected student at some point in the future you'll need to do one of two things:
1. Move this logic out of its own component into the Details component
2. Add an onChange callback prop that gets called each time the selection is changed.

I think the first one would be easier. Let me know if you have questions about this.

File: public/scripts/models/StoryModel.js [hunk 4]
```diff
@@ -8,7 +8,8 @@ export default Backbone.Model.extend({
 		title: '',
 		author: '',
 		category: '',
-		coverImage: ''
+		coverImage: '',
+		description: ''
 	},
 	urlRoot: '/api/v1/story',
 	idAttribute: 'id'
```
Comment (alarner) [inline public/scripts/models/StoryModel.js:15]: The StoryModel shouldn't have changed for this ticket, please undo these changes.

**Comment by NicerHugs** [comment]
1. pls fix merge conflicts before pr can be approved
2. usually it's a bad idea to edit a migration file directly, instead you would create a new migration file and make the changes to the table there. Tho maybe this was discussed and planned and therefore ok? 
3. if the migration thing is not an issue, this looks good, just remove `console.log`s before pr can be approved

**Comment by judan** [comment]
Can everyone give this another look? I just resolved the merge conflicts and removed my console logs. Please and thank you!!

**Comment by NicerHugs** [comment]
Is this component meant to include updating something on the details about the student that was selected? If so, that code needs to be incorporated. If that is on a later ticket, then 👍

**Comment by judan** [comment]
I just moved everything from that subcomponent file into the Details page. I think I'm done now? For realsies this time. Plz gimme thumbz so I can work on another ticket. Thanks!

File: public/scripts/components/pages/Details.js [hunk 5]
```diff
@@ -1,10 +1,37 @@
...
+	componentDidMount: function() {
+		students.on('update', () => {
+			this.setState({students: students});
+		});
+		students.fetch();
+	},
 	render: function() {
+		const allStudents = this.state.students.map((student, index, array) => {
+		return (
+			<option value="Student" key={index}>{student.get('firstName')} {student.get('lastName')}</option>
```
Comment (alarner) [inline public/scripts/components/pages/Details.js]: The value here should be the id of the student, so that when we click read now we can get the currently selected student. After that I think your done @judan !

**Comment by DanBuda11** [comment]
👍

**Comment by judan** [comment]
yay i did it! dunzo! (can i merge??) 🎱

**Comment by NicerHugs** [comment]
👍
